app/static/css/styles.css

```diff
@@ -11,8 +11,24 @@
 
 /*General*/
 body {
-    margin: 0;
     padding: 0;
+    margin-bottom: 60px;
+}
+
+html {
+    position: relative;
+    min-height: 100%;
+    padding-bottom: 60px;
+}
+
+.footer {
```

Comment:
There is already a footer class in this file. Please be on the lookout for unnecessary bloat and maintenance burden. This will create a debugging nightmare if we merge something like this with two implementations that diverge over time.

Reply:
@friedelwolff apologies for that, that was meant to be deleted. I'll be more careful to review my code before its sent for review